src/client/app/components/DashboardComponent.jsx

```diff
@@ -12,12 +12,22 @@ import { chartTypes } from '../reducers/graph';
  * React component that controls the dashboard
  */
 export default function DashboardComponent(props) {
+	const divPadding = {
+		paddingTop: '35px'
+	};
+	const ChartToRender = (props.chartToRender === chartTypes.line) ? LineChartContainer : BarChartContainer;
 	return (
 		<div className="container-fluid">
-			<UIOptionsContainer />
-			<div className="col-xs-10">
-				{props.chartToRender === chartTypes.line ? <LineChartContainer /> : <BarChartContainer />}
-			</div>
+			{ props.drawUIOptions ? (
```

Comment:
This is identical to how the React docs do it. See https://facebook.github.io/react/docs/conditional-rendering.html, under the sectionInline If with Logical && Operator